app/views/spree/admin/variants/_form.html.haml

```diff
@@ -98,7 +97,7 @@
       = f.collection_select(:primary_taxon_id, Spree::Taxon.order(:name), :id, :name, { include_blank: true }, { class: "select2 fullwidth" })
 
     .field
-      = f.label :supplier, t(:spree_admin_supplier)
+      = f.label :supplier, raw(t(:spree_admin_supplier) + content_tag(:span, ' *', :class => 'required'))
```

Comment:
I'm not sure, but the formbuilder might add the asterisk for us?
```suggestion
      = f.label :supplier, t(:spree_admin_supplier), required: true
```

But if not, perhaps a block will allow us to avoid the `raw` call:
```suggestion
      = f.label :supplier do
        = t(:spree_admin_supplier)
        = content_tag(:span, ' *', :class => 'required'))
```

Reply:
I'll have a look, I think I copied that from somewhere else